src/bin/juliainstaller.rs

```diff
@@ -379,6 +379,35 @@ pub fn main() -> Result<()> {
         }
     }
 
+    if install_choices.install_location.exists() {
+        println!("You are trying to install Juliaup into the folder");
+        println!("`{}`,", install_choices.install_location);
+        println!("but that folder already exists.");
+
+        if args.disable_confirmation_prompt {
+            println!();
+            println!(
+                "Please remove the folder or use interactive mode."
+            );
+
+            return Ok(());
+        } else {
+            let continue_with_setup = Confirm::with_theme(theme.as_ref())
+                .with_prompt("Do you want to continue with the installation and overwrite the existing Juliaup installation folder?")
+                .default(true)
+                .interact_opt()?;
+
+            if !continue_with_setup.unwrap_or(false) {
+                return Ok(());
+            }
+
+            std::fs::remove_dir_all(install_choices.install_location)
```

Comment:
Or maybe this is too drastic? A user might have picked a folder where other (legit) stuff is, and then we delete it all. Probably not good... Maybe we just output an error message saying: "select a folder that doesn't exist" or something like that?